src/theme.js

```diff
@@ -1,37 +1,99 @@
-import { purple, green, red } from './colors'
+const baseColors = {
+  mono: {
+    black: '#000',
+    nearblack: '#262626',
+    darkgray: '#666666',
+    lightgray: '#C4C4C4',
+    silver: '#999999',
+    white: '#FFF',
+    transparent: 'transparent'
+  },
+  blue: {
+    lightest: '#F6F8FE',
+    lighter: '#EFF3FD',
+    light: '#E4EBFC',
+    mid: '#d1ddfa'
+  },
+  green: {
+    primary: '#1AD08F',
+    light: '#D2F5ED',
+    dark: '#007056'
+  },
+  red: {
+    light: '#FC6D6F',
+    dark: '#660000'
+  },
+  yellow: {
+    light: '#FFDC99',
+    deep: '#FCA703'
+  },
+  purple: {
+    light: '#E0D7FE',
+    deep: '#5E26FF'
+  }
+}
 
 // colors
 const colors = {
-  text: '#262626',
-  background: '#F6F8FE',
-  purple: purple.base,
-  lightpurple: purple.text,
-  black: '#000',
-  nearBlack: '#111',
-  darkGray: '#333',
-  midGray: '#555',
-  silver: '#999',
-  'light-silver': '#aaa',
-  'moon-gray': '#ccc',
-  'light-gray': '#eee',
-  'near-white': '#f4f4f4',
-  white: '#fff',
-  transparent: 'transparent',
-  success: {
-    base: green.base,
-    text: green.text
+  // The core color array is the only non-semantic array here. This is because these colors are used so widely that it would be highly redundant to replicate these color values repeatedly within this "colors" object to style the text/background of every single Component the app renders.
+  core: {
```

Comment:
@Schwartz10 FYI - Most of the `core` colors pull down from the `baseStyles` `mono` array. 

It seems redundant to duplicate them here. Why do it? My thinking is:

1. That this `colors` array should always cite `baseColors` for its values to establish a clear hierarchy
1. If we ever need to change the value of something like `darkgray` or `lightgray` we can do so inside of `baseColors` without needing to edit the `colors` array and thus break any components using them.
1. That the `core` colors in the `colors` array will be repeatedly used, and so are better organized non-semantically (everything else is defined semantically) rather than having lots of single-use semantic color objects